src/pages/tutorials/library.tsx

```diff
@@ -38,7 +38,8 @@ const DEFAULT_SLUGS = [
 ]
 
 const appId = process.env.NEXT_PUBLIC_ALGOLIA_APP_ID
-const searchClient = algoliasearch(appId, 'bf27a047ba263cba01ee9b4081965a1a')
+const apiKey = process.env.NEXT_PUBLIC_ALGOLIA_SEARCH_ONLY_API_KEY
+const searchClient = algoliasearch(appId, apiKey)
```

Comment:
This update was needed since the inline read-only Algolia API key here didn't have permission to access `prod_DEVDOT_omni`.